Core/Handlers/MessageHandler.php

```diff
@@ -29,6 +29,7 @@
 use Smartbox\Integration\FrameworkBundle\DependencyInjection\Traits\UsesEndpointFactory;
 use Symfony\Component\DependencyInjection\ContainerAwareInterface;
 use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+use  Smartbox\Integration\FrameworkBundle\Core\Messages\CallbackExchangeEnvelope;
```

Comment:
Here you can an extra space. I suggest to pass the php cs fixer

Reply:
I did run it, but it didn't remove that :-D